### DISCUSSION TITLE
Trial Account not functioning

### DISCUSSION THREAD

**Discussion by frazer11826** [opengisch/QField#4501] [Bugs]
Have signed up for a trial account, but it states that the active until date has passed. Happy to pay now if that resolves things but need to get some work setup for next week if possible.

![image](https://github.com/opengisch/QField/assets/69894337/a5266786-be9c-4b79-9ca2-16904a0b806b)

**Comment by faebebin** [comment]
Hi, @frazer11826 there is a fix in the pipeline for trial organizations to be released soon. But this discussion would rather belong to https://github.com/opengisch/qfieldcloud . 

For specific issues with trial organizations on qfieldcloud please create a ticket on https://app.qfield.cloud/ (the `?` button on bottom right) adding the name of the organization so we can fix it. 

Thanks
